internal/evaluation_target/application_snapshot_image/certificate.go

```diff
@@ -0,0 +1,153 @@
+	"Subject":                                   nameFrom(func(c *x509.Certificate) pkix.Name { return c.Subject }),
+	"Subject Alternative Name":                  san,
+	"Issuer":                                    nameFrom(func(c *x509.Certificate) pkix.Name { return c.Issuer }),
+	"Serial Number":                             func(c *x509.Certificate) (string, error) { return c.SerialNumber.Text(16), nil },
+	"Not Before":                                func(c *x509.Certificate) (string, error) { return c.NotBefore.UTC().Format(time.RFC3339), nil },
+	"Not After":                                 func(c *x509.Certificate) (string, error) { return c.NotAfter.UTC().Format(time.RFC3339), nil },
+	"Fulcio Issuer":                             rawString(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 1}),
+	"Fulcio GitHub Workflow Trigger":            rawString(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 2}),
+	"Fulcio GitHub Workflow SHA":                rawString(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 3}),
+	"Fulcio GitHub Workflow Name":               rawString(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 4}),
+	"Fulcio GitHub Workflow Repository":         rawString(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 5}),
+	"Fulcio GitHub Workflow Ref":                rawString(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 6}),
+	"Fulcio Issuer (V2)":                        utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 8}),
+	"Fulcio Build Signer URI":                   utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 9}),
+	"Fulcio Build Signer Digest":                utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 10}),
+	"Fulcio Runner Environment":                 utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 11}),
+	"Fulcio Source Repository URI":              utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 12}),
+	"Fulcio Source Repository Digest":           utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 13}),
+	"Fulcio Source Repository Ref":              utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 14}),
+	"Fulcio Source Repository Identifier":       utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 15}),
+	"Fulcio Source Repository Owner URI":        utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 16}),
+	"Fulcio Source Repository Owner Identifier": utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 17}),
+	"Fulcio Build Config URI":                   utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 18}),
+	"Fulcio Build Config Digest":                utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 19}),
+	"Fulcio Build Trigger":                      utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 20}),
+	"Fulcio Run Invocation URI":                 utf8String(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 21}),
```

Comment:
It would be nice to add a comment with a link to the source where all these are documented. (Assuming such a think exists.)